### ISSUE TITLE
重复任务优化 失败

### ISSUE THREAD

**Issue by icelove888** [6dylan6/jdpro#311]
## 开始执行... 2022-06-25 21:06:02

===> 禁用重复任务开始 <===
加载推送功能中...

=== 你选择过滤的任务前缀为 ===
6dylan6_jdpro
❌无法获取 tasklist!!!

## 执行结束... 2022-06-25 21:06:03 耗时 1 秒

**Comment by icelove88** [comment]
v2.13.3

**Comment by icelove88** [comment]
没有 .默认 5700

**Comment by icelove88** [comment]
对.  开始提示端口 错误. 我加了  export IPPORT='127.0.0.1:5700'  这个就提示这个错误.

**Comment by icelove88** [comment]
## 开始执行... 2022-06-22 21:22:31

如果报错请在环境变量中添加你的真实 IP:端口
名称：IPPORT	值：127.0.0.1:5700
或在 config.sh 中添加 export IPPORT='127.0.0.1:5700'
===> 禁用重复任务开始 <===
加载推送功能中...

=== 你选择过滤的任务前缀为 ===
6dylan6_jdpro
❌无法获取 tasklist!!!

## 执行结束... 2022-06-22 21:22:32 耗时 1 秒


这是第一次错误提示

**Comment by kbill4345** [comment]
> 
不管加不加export IPPORT='127.0.0.1:5700'，一样的问题，qinglong是2.12.1

**Comment by blusunny** [comment]
解决了吗？ 也遇到了，2.15.8版本青龙
